### PR TITLE
[FR] Update `BBCode`

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- (TicClick) skipped: Add #9164 changes
- (TicClick) skipped: Add #9263 changes
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/BBCode/fr.md [hunk 1]
```diff
@@ -148,8 +143,22 @@ Les citations longues sont généralement utilisées dans des écrits plus forme
...
 
+### Code en ligne
+
+*Ne pas confondre avec [Bloc de codes](#bloc-de-codes).*
+
+```
+[c]text[/c]
+```
+
+La balise `[c]` permet de mettre en évidence du texte en ligne dans une police monospace. Sur le site d'osu!, le texte sera formaté avec une boîte grise autour de lui. contrairement aux [blocs de codes](#bloc-de-codes), la balise ne peut être placée que sur une seule ligne.
```
Comment (Pisapou) [inline wiki/BBCode/fr.md:154]:
```suggestion
La balise `[c]` permet de mettre en évidence du texte en ligne dans une police monospace. Sur le site d'osu!, le texte sera formaté avec une boîte grise autour de lui. Contrairement aux [blocs de codes](#bloc-de-codes), la balise ne peut être placée que sur une seule ligne.
```

File: wiki/BBCode/fr.md [hunk 2]
```diff
@@ -148,8 +143,22 @@ Les citations longues sont généralement utilisées dans des écrits plus forme
...
+
+Dans les forums d'osu!, il est utilisé afin de pouvoir mettre des accents, par exemple sur les raccourcis clavier ou les descriptions de boutons.
```
Comment (Pisapou) [inline wiki/BBCode/fr.md:156]:
```suggestion
Dans les forums d'osu!, il est utilisé afin de mettre en avant, par exemple des raccourcis clavier ou des descriptions de boutons.
```

**Review by Pisapou — CHANGES_REQUESTED** [review]

File: wiki/BBCode/fr.md [hunk 3]
```diff
@@ -148,8 +143,22 @@ Les citations longues sont généralement utilisées dans des écrits plus forme
...
+*Ne pas confondre avec [Bloc de codes](#bloc-de-codes).*
+
+```
+[c]text[/c]
+```
+
+La balise `[c]` permet de mettre en évidence du texte en ligne dans une police monospace. Sur le site d'osu!, le texte sera formaté avec une boîte grise autour de lui. Contrairement aux [blocs de codes](#bloc-de-codes), la balise ne peut être placée que sur une seule ligne.
+
+Dans les forums d'osu!, il est utilisé afin de mettre en avant, par exemple des raccourcis clavier ou des descriptions de boutons.
+
```
Comment (The-Last-Cookie) [inline wiki/BBCode/fr.md:157]: Changes from #9263:
```suggestion

Toolbar button: ![Imagemap button](img/imagemap.png "Imagemap")

```

**Comment by The-Last-Cookie** [comment]
If you want to add a localised version for `editor.jpg`, keep in mind translating the new `Help` string in the toolbar via Crowdin.

**Review by TicClick — APPROVED** [review]
marking as outdated due to https://github.com/ppy/osu-wiki/pull/9164 and merging as-is -- it's been well over a month since the PR was posted

**Pull request merged**